src/JuliusSweetland.OptiKey/UI/ViewModels/MainViewModel.ServiceEventHandlers.cs

```diff
@@ -2045,5 +1963,85 @@ private void HandleServiceError(object sender, Exception exception)
                 audioService.PlaySound(Settings.Default.ErrorSoundFile, Settings.Default.ErrorSoundVolume);
             }
         }
+
+        private void NavigateToMenu()
+        {
+            Log.Info("Changing keyboard to Menu.");
+            Keyboard = new Menu(CreateBackAction());
+        }
+
+        private void NavigateToVoiceKeyboard()
+        {
+            List<string> voices = GetAvailableVoices();
+
+            if (voices != null && voices.Count > 0)
```

Comment:
&& voices.Any()

Reply:
Thanks. Always forget about Any. Will update it here and the other place you mentioned as well.